src/crimson/osd/objclass.cc

```diff
@@ -48,11 +48,16 @@ int cls_read(cls_method_context_t hctx,
              char **outdata,
              int *outdatalen)
 {
+  ceph_assert("not implemented yet" == nullptr);
   return 0;
 }
 
 int cls_get_request_origin(cls_method_context_t hctx, entity_inst_t *origin)
 {
+  if (origin) {
```

Comment:
shall we instead `assert(origin)` ?

Reply:
Yes, the latter https://github.com/ceph/ceph/pull/30339/commits/7ef1161deb3c2f21cfb0545ca73e514affed69c5 does that. Fixing the older commit as well.

Reply:
Fixed.